RioCode-Old/src/main/java/frc/robot/Commands/StateCommands/CoralReefPose.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.Commands.StateCommands;
+
+import org.ejml.dense.row.decomposition.hessenberg.TridiagonalDecompositionHouseholderOrig_FDRM;
+
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
+import frc.robot.TriggerBoard;
+import frc.robot.subsystems.RobotMode;
+
+public class CoralReefPose extends Command{
+    
+    public CoralReefPose() {
+        addRequirements(Robot.arm, Robot.climber, Robot.intake, Robot.manipulator, Robot.elevator);
+    }
+
+    @Override
+    public void initialize() {
+        if (TriggerBoard.isL1Selected()) {
+
+        } else if (TriggerBoard.isL2Selected()) {
+
+        } else if (TriggerBoard.isL3Selected()) {
+
+        } else if (TriggerBoard.isL4Selected()) {
+
+        } else {
+            Robot.joystick.setRumble(RumbleType.kBothRumble, 1);
+            Robot.joystick.setRumble(RumbleType.kBothRumble, 0);
```

Comment:
Same here, recommend a single function to denote an error with some text, that way if the driver is curious what the rumble was for, they can look at the logs.